mod/directory.php

```diff
@@ -62,7 +62,7 @@ function directory_content(App $a)
 	if ($alpha) {
 		$order = ' ORDER BY `name` ASC ';
 	} else {
-		$order = ' ORDER BY `updated` DESC, `id` DESC ';
+		$order = ' ORDER BY `filled_fields` DESC, `last_activity` DESC, `updated` DESC';
```

Comment:
Wouldn't it be better if the sorting would be by the month of the last update and then the number of filled fields? Sorting after the filled fields at first could put someone at first who had filled out all fields - but hadn't been active for years.

Reply:
The last_actitivy is `YYYY-WW` with WW being the week number in the year, so we can't sort on the month if it made any sense. Besides, this sorting is only to improve the visual impression of the default display of the directory. The first page will always have filled profiles with recent activity. What happens in the subsequent pages is way less relevant.

Reply:
Ah, yeah. We already compacted the data. So possibly we should sort by "last_activity" and "filled_fields" instead.

Reply:
And then you'd have all the spam bots without any profile info first. Believe me, I thought about it, I tried different orders, and this was the most visually gratifying.

Reply:
Okay. Since we can change that very fast (and since there aren't that many installations), we can test it out.